autospec/autospec.py

```diff
@@ -175,6 +178,14 @@ def main():
             "-a/--archives or options.conf['package']['archives'] requires an "
             "even number of arguments"))
 
+    if args.prep_only:
+        package(args, url, name, archives, "./workingdir")
```

Comment:
I see an empty directory called "output" within workingdir. Is that directory supposed to be empty?

Reply:
I've removed the unused output directory